{{cookiecutter.project_slug}}/Makefile

```diff
@@ -116,14 +159,18 @@ clean-pyc: ## remove Python file artifacts
 	find . -name '*~' -exec rm -f {} +
 	find . -name '__pycache__' -exec rm -fr {} +
 
-.PHONY: clean-test
-clean-test: ## remove test and coverage artifacts
-	rm -fr .tox/
+.PHONY: clean-coverage
+clean-coverage: ## remove coverage artifacts
 	rm -f .coverage
+	rm -f .coverage.*
```

Comment:
Yes, it is. In some scenarios coverage creates intermediate reports called `.coverage.${some_hash}`, which then are collected and removed by the `coverage report` command.

I don't remember the exact scenario, and I cannot reproduce it now either, but I remember having added this to BTB because if you pressed Ctrl-C in the middle of the `make coverage` run there were files left there that were not properly cleaned.

Reply:
sounds good